keras_cv/models/segmentation/deeplabv3plus.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+
+from keras_cv.layers.spatial_pyramid import SpatialPyramidPooling
+from keras_cv.models import utils
+from keras_cv.models.weights import parse_weights
+
+
+@keras.utils.register_keras_serializable(package="keras_cv")
+class DeepLabV3Plus(keras.Model):
+    """
+    A segmentation model based on the DeepLabV3Plus model.
+
```

Comment:
paper reference.

Reply:
Updated - we were missing it in DeepLabV3 too.